packages/realm-web/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "realm-web",
+  "version": "3.6.4",
+  "description": "Authenticate and communicate with the MongoDB Realm platform, from your web-browser",
+  "main": "dist/index.js",
+  "types": "types/index.d.ts",
+  "scripts": {
+    "prebuild": "rm -rf dist",
```

Comment:
I'll remove it, it's was more to remove the annoyance of importing stale code when moving a file from `some-service.js` to `some-service/index.js` and not importing the correct file because `some-service.js` was still in the dist folder from a previous build.

Reply:
:+1